list2/ex21/index.js

```diff
@@ -0,0 +1,42 @@
+import promptSync from 'prompt-sync';
+const prompt = promptSync();
+
+const WOMEN_WEIGHT_IDEAL_BASE = 62.1;
+const WOMEN_WEIGHT_FORMULA = 44.7;
+const MEN_WEIGHT_IDEAL_BASE = 72.7;
+const MEN_WEIGHT_FORMULA = 58;
+
+
+export function idealWeightFunction() {
+    const gender = {
+        male: 'M',
+        female: 'F'
+    }
+
+    console.log('What is the person gender? (Use M or F).');
+    const genderPrompt = prompt('>').toUpperCase();
+
+    if(!Object.values(gender).includes(genderPrompt)) {
+        console.log('Insert a valid gender.');
+        return;
+    }
+
+    console.log('What is the person height in meters ? (Use just float numbers).');
+    const heightPrompt = parseFloat(prompt('>'));
+
+    function idealWeightCalculation(height) {
+        let idealWeight;
+
+        if (genderPrompt === gender.male) {
```

Comment:
The way you used the if condition is totally fine, but I'd suggest practicing with other possible ways as well. Using `switch` is one of them.

Reply:
I'll keep this in mind for future exercises